tools/gen_ort_dockerfile.py

```diff
@@ -281,9 +285,9 @@ def dockerfile_for_linux(output_file):
         ep_flags += (
             " --skip_tests --cmake_extra_defines 'onnxruntime_BUILD_UNIT_TESTS=OFF'"
         )
-        cuda_archs = "53;62;72;87"
+        cuda_archs = "87;101"
```

Comment:
I'm afraid we may still support Xavier, which code `72`, need to check it.
```suggestion
        cuda_archs = "72;87;101"
```